frontend/src/lib/components/shared/Dropdown/Selectbox.tsx

```diff
@@ -95,6 +99,8 @@ class Selectbox extends React.PureComponent<Props, State> {
   private onChange = (params: OnChangeParams): void => {
     if (params.value.length === 0) {
       logWarning("No value selected!")
+      this.setState({ value: undefined })
```

Comment:
Are you sure that this isn't introducing any issues? In which sitations is this triggered with `params.value.length === 0`? Only when the user clears the value?

Reply:
I do not think so, just checked both `st.multiselect` and `st.selectbox` with empty lists/tuples and everything works fine, however we can double check with @jrieke